scripts/batch.py

```diff
@@ -0,0 +1,328 @@
+"""Batch Processing Script.
```

Comment:
You can just start with the 2nd line explaining what the script does. The name is self-explanatory.